src/__swaps__/screens/Swap/components/ReviewPanel.tsx

```diff
@@ -195,7 +183,7 @@ export function ReviewPanel() {
             </Inline>
 
             <Inline wrap={false} horizontalSpace="8px" alignVertical="center">
-              <ButtonPressAnimation onPress={() => onSetSlippage('decrement')}>
+              <ButtonPressAnimation onPress={() => runOnUI(SwapSettings.onUpdateSlippage)('minus')}>
```

Comment:
i think you can use `GestureHandlerV1Button` so the onPress is automatically executed on the UI thread and you don't need to bridge over from JS

Reply:
this applies to some other places too